policyengine_us/parameters/gov/states/ga/tax/income/deductions/standard/aged/age_eligible.yaml

```
@@ -0,0 +1,14 @@
+description: Georgia provides additional standard deduction for the taxpayer or spouse at or above the following age.
+metadata:
+  label: Georgia additional standard deduction age eligibility
```

Also rename the file `age_threshold.yaml`
```suggestion
  label: Georgia additional standard deduction age threshold
```